ciao-launcher/main.go

```diff
@@ -341,6 +343,54 @@ func printClusterConfig() {
 	glog.Infof("Memory Limit:         %v", memLimit)
 }
 
+func discoverCertificate(certPath, certPrefix string) (string, error) {
+	ciaoCertsPath := "/etc/pki/ciao/"
+	fmt.Println("discoverCertificate")
```

Comment:
did you mean to log here? we typically do not perform direct printing. you always use --alsologtostderr